app/src/main/java/com/github/se/polyfit/ui/screen/IngredientsOverview.kt

```diff
@@ -42,16 +47,32 @@ fun IngredientsOverview(
     navigateBack: () -> Unit,
     navigateForward: () -> Unit,
     onClickFloatingButton: () -> Unit,
-    listProducts: List<IngredientsTMP>
+    listProducts: List<IngredientsTMP>,
+    barCodeCodeViewModel: BarCodeCodeViewModel = hiltViewModel()
 ) {
   val context = LocalContext.current
 
+  val listId by barCodeCodeViewModel.listId.observeAsState()
+
+  // TODO REMOVE
+  val listProducts = mutableListOf<IngredientsTMP>()
+  for (qrCode in listId ?: emptyList()) {
+    listProducts += IngredientsTMP(qrCode, 0, 0, 0, 0, 0)
+  }
+
   Scaffold(
       topBar = { SimpleTopBar(title = context.getString(R.string.Product)) { navigateBack() } },
       bottomBar = { BottomBarIngredient(navigateForward = navigateForward) },
       floatingActionButton = { FloatingActionButtonIngredients(onClickFloatingButton) },
       containerColor = MaterialTheme.colorScheme.background,
       modifier = Modifier.testTag("IngredientsOverviewScaffold")) {
-        ListProducts(listIngredients = listProducts, Modifier.fillMaxSize().padding(it))
+        CameraXScreen(barCodeCodeViewModel)
+
+        // TODO THIS IS TEMPORARY, REMOVE IT WHEN ADDING THE REAL VALUES
+        ListProducts(
```

Comment:
I do not know if this is in the scope of this PR or maybe a future one, but it would be great if you could remove an ingredient from the list if you add it by accident, similar to how it works in the ingredient screen when creating a meal.